plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/product-collection-content.tsx

```diff
@@ -73,14 +72,14 @@ const ProductCollectionContent = ( {
 }: ProductCollectionEditComponentProps ) => {
 	const isInitialAttributesSet = useRef( false );
 	const { clientId, attributes, setAttributes } = props;
```

Comment:
```suggestion
	const { clientId, attributes, location, productCollectionUIStateInEditor, setAttributes } = props;
```

We can destructure all props here and remove next line and simply line 82. (Apologies for such instructions and not code example but GitHub didn't allow me to provide suggestion to the whole section due to removed lines 😅 )

Reply:
No problem at all 😀 I will make the change. Thanks for the suggestion. 🙌🏻